golos.vesting/golos.vesting.cpp

```diff
@@ -65,7 +65,7 @@ void vesting::on_transfer(name from, name to, asset quantity, std::string memo)
     asset converted = token_to_vesting(quantity, *vesting, -quantity.amount);
     table_vesting.modify(vesting, name(), [&](auto& item) {
         item.supply += converted;
-        // TODO Add notify about supply changes #548
+        send_supply_event(item.supply);
```

Comment:
It's better to pass in `send_supply_event` function all item. `send_supply_event` get `supply` field and send it in notify. So if we need send more fields in feature, only one place need to be fixed.